src/Components/SuperAdmins/Form/index.jsx

```diff
@@ -61,10 +72,17 @@ const Form = () => {
       });
       const data = await response.json();
       if (response.ok) {
-        alert(data.message);
-        handleCancel();
+        setTypeStyle('success');
+        setTitleModal('Success');
+        setBodyModal('Super Admin updated successfully.');
+        setShowSuccessModal(true);
+        setShowErrorModal(false);
       } else {
-        alert(data.message);
+        setTypeStyle('error');
+        setTitleModal('Error');
+        setBodyModal(data.message);
+        setShowModal(true);
+        setShowSuccessModal(false);
       }
     } catch (error) {
       console.error(error);
```

Comment:
I would recommend not to leave any console.log in the final code

Reply:
no is a console.log.. is .error!